docs/users.md

```diff
@@ -0,0 +1,87 @@
+     ...
+     ```
+1. Update the value of `metadata.name` with any name for this secret.
+1. Update the value of `stringData.password` with the user's password.
+1. Save the secret with a `.yaml` file extension.
+1. Apply the secret in Kubernetes:
+   ```
+   kubectl apply -f <db-user-secret>.yaml --namespace <my-namespace>
+   ```
+
+## Modify the MongoDB CRD
+
+1. Add the following fields to the MongoDB resource definition:
+
+   | Key | Type | Description | Required? |
+   |----|----|----|----|
+   | `spec.users` | array of objects | Configures database users for this deployment. | Yes |
+   | `spec.users.name` | string | Username of the database user. | Yes |
+   | `spec.users.db` | string | Database that the user authenticates against. Defaults to `admin`. | No |
+   | `spec.users.passwordSecretRef.name` | string | Name of the secret that contains the user's plain text password. | Yes|
+   | `spec.users.passwordSecretRef.key` | string| Key in the secret that corresponds to the value of the user's password. Defaults to `password`. | No |
+   | `spec.users.roles` | array of objects | Configures roles assigned to the user. | Yes |
+   | `spec.users.roles.role.name` | string | Name of the role. Valid values are [built-in roles](https://docs.mongodb.com/manual/reference/built-in-roles/#built-in-roles). | Yes |
+   | `spec.users.roles.role.db` | string | Database that the role applies to. | Yes |
+
+   ```
+   ---
+   apiVersion: mongodb.com/v1
+   kind: MongoDB
+   metadata:
+     name: example-scram-mongodb
+   spec:
+     members: 3
+     type: ReplicaSet
+     version: "4.2.6"
+     security:
+       authentication:
+         modes: ["SCRAM"]
+     users:
+       - name: <username>
+         db: <authentication-database>
+         passwordSecretRef: 
+           name: <db-user-secret>
+         roles:
+           - name: <role-1>
+             db: <role-1-database>
+           - name: <role-2>
+             db: <role-2-database>
+   ...
+   ```
+1. Save the file.
+1. Apply the updated MongoDB resource definition:
+
+   ```
+   kubectl apply -f <mongodb-crd>.yaml --namespace <my-namespace>
+   ```
+1. After the MongoDB resource is running, securely store the user's password and then delete the user secret:
+   ```
+   kubectl delete secret <db-user-secret> --namespace <my-namespace>
+   ```
+
+## Next Steps
+
+- To authenticate to your MongoDB resource, run the following command:
+   ```
+   mongo "mongodb://<mongodb-resource-metadata.name>-svc.<my-namespace>.svc.cluster.local:27017/?replicaSet=<replica-set-name>" --username <username> --password <password> --authenticationDatabase <authentication-database>
+   ```
+- To change a user's password, create and apply a new secret with a `metadata.name` that is the same as the name specified in `passwordSecretRef.name` of the MongoDB CRD. The Operator automatically re-generates the SCRAM credentials.
```

Comment:
[nit] `create a secret` already indicates that the kubernetes secret resource is being created, but here you are referring to creating a new resource definition that contains the yaml for the new password.

I think we should phrase this as `create and apply a new secret resource definition` or just `create a new secret as described above`

Reply:
[tiny-nit] The line `The Operator automatically re-generates the SCRAM credentials.` sounds a little off to me, I think we can just say, `The Operator will automatically generate new credentials`

Reply:
Gotcha. Updated.